src/page/home/community/community.tsx

```diff
@@ -0,0 +1,118 @@
+  <div className="okp4-dataverse-portal-home-page-stat-main">
+    {icon && <div className="okp4-dataverse-portal-home-page-stat-icon-container">{icon}</div>}
+    <div className="okp4-dataverse-portal-home-page-stat-content-wrapper">
+      <h2>{stat}</h2>
+      <p>{description}</p>
+    </div>
+  </div>
+)
+
+// eslint-disable-next-line max-lines-per-function
+export const Community: FC<CommunityProps> = ({ className }) => {
+  const { t } = useTranslation('home')
+  const theme = useAppStore(store => store.theme)
+  const { dataspacesCreatedNb, datasetsSharedNb, servicesSharedNb, communityParticipantNb } =
+    useStatsAggregate(
+      aggregate => ({
+        dataspacesCreatedNb: aggregate.dataspacesCreatedNb,
+        datasetsSharedNb: aggregate.datasetsSharedNb,
+        servicesSharedNb: aggregate.servicesSharedNb,
+        communityParticipantNb: aggregate.communityParticipantNb
+      }),
+      shallow
+    )
+
+  const statCards: StatCard[] = useMemo(
+    () => [
+      {
+        _tag: 'singlestat',
+        stat: communityParticipantNb,
+        description: t('home.blocks.community.stats.communityParticipants')
+      },
+      {
+        _tag: 'multistat',
+        stats: [
+          {
+            icon: <Icon name={`dataspace-created-${theme}`} />,
+            stat: dataspacesCreatedNb,
+            description: t('home.blocks.community.stats.dataspacesCreated')
+          },
+          {
+            icon: <Icon name={`dataset-folder-${theme}`} />,
+            stat: datasetsSharedNb,
+            description: t('home.blocks.community.stats.datasetsShared')
+          },
+          {
+            icon: <Icon name={`service-folder-${theme}`} />,
+            stat: servicesSharedNb,
+            description: t('home.blocks.community.stats.servicesShared')
+          }
+        ]
+      }
+    ],
+    [communityParticipantNb, datasetsSharedNb, dataspacesCreatedNb, servicesSharedNb, t, theme]
+  )
+
+  return (
+    <div className={`okp4-dataverse-portal-home-page-community-main ${className}`}>
+      <h1>{t('home.blocks.community.label')}</h1>
+      {statCards.map(statCard => (
+        <Card
+          className={`okp4-dataverse-portal-home-page-community-${statCard._tag}-card`}
+          key={statCard._tag}
+        >
+          <div className={`okp4-dataverse-portal-home-page-community-${statCard._tag}-container`}>
+            {statCard._tag === 'multistat' ? (
+              statCard.stats.map(({ description, stat, icon }, index) => (
+                <Stat description={description} icon={icon} key={index} stat={stat} />
+              ))
+            ) : (
+              <Stat
+                description={statCard.description}
+                icon={statCard.icon}
+                key={statCard._tag}
+                stat={statCard.stat}
+              />
+            )}
+            {statCard._tag === 'singlestat' && (
+              <div className="okp4-dataverse-portal-home-page-community-ellipse" />
```

Comment:
I think you can move the `ellipse` directly in the second part of the ternary, what do you think?

Reply:
done 👍